
I have a PowerPC board with Cirrus chip here, the driver compiles - no idea
it it works out-of-the-box ...

  - Rene Rebe <rene@exactcode.de>

--- ./config/cf/xorg.cf.orig	2005-01-17 18:13:00.730993000 +0100
+++ ./config/cf/xorg.cf	2005-01-17 18:11:52.121423240 +0100
@@ -619,7 +619,7 @@
 /* Pure PCI drivers should go first */
 # ifndef XF86CardDrivers
 #  define XF86CardDrivers	ati mga glint nv s3 s3virge sis savage\
-				trident chips tdfx \
+				cirrus trident chips tdfx \
 				DevelDrivers vga \
 				XF86OSCardDrivers XF86ExtraCardDrivers
 # endif
